### PR TITLE
Added Interesting Pet Names.

### PR DESCRIPTION
**Pull request by ShabanaNaik**
Hello @tonybaloney,
After a lot of interesting name searches, I came upon the Star buck's, Pokémon, Disney characters and many common names for Dog, Snake, Rocky, Zappy and Duck names.
I have added names that seemed logical or interesting based on the pet's visuals present in the extension.
And thank you for the extension I have been using it for a few days now when I feel tried or irritated while coding 😣. And the pets really made me feel at ease after playing with them😁😊. Especially the snake and duck. I really liked the whole snake on a cart (or 2 wheels) and duck with a jet pack and radio on it back with the sunglasses look 🤩.

### CODE REVIEW

File: src/common/names.ts [hunk 1]
```diff
@@ -338,6 +358,30 @@ export const DUCK_NAMES: Map<number, string> = new Map<number, string>([
...
+    [19, 'Mr.Quack'],
+    [20, 'Louie'],
+    [21, 'Golduck'],
+    [22, 'Daisy'],
+    [23, 'Pickles'],
+    [24, 'Ducky Duck'],
+    [25, 'Mrs Fluffs'],
+    [26, 'Squeek'],
+    [27, 'Ace'],
+    [28, 'Little Rey'],
```
Comment (tonybaloney) [inline src/common/names.ts:372]: Please remove the star wars references ( I don't want to be sued by Disney !)

File: src/common/names.ts [hunk 2]
```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
...
     [9, 'Max'],
     [10, 'Seb'],
+    [11, 'Mr.Hiss'],
+    [12, 'Miss.Hiss'],
+    [13, 'Snaku'],
+    [14, 'Kaa'],
+    [15, 'Madame Snake'],
+    [16, 'Sir Hiss'],
+    [17, 'My Python'],
+    [18, 'The Anaconda'],
```
Comment (tonybaloney) [inline src/common/names.ts:330]: Please remove "The Anaconda"

File: src/common/names.ts [hunk 3]
```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
     [8, 'Bailey'],
     [9, 'Max'],
     [10, 'Seb'],
+    [11, 'Mr.Hiss'],
+    [12, 'Miss.Hiss'],
+    [13, 'Snaku'],
+    [14, 'Kaa'],
+    [15, 'Madame Snake'],
+    [16, 'Sir Hiss'],
+    [17, 'My Python'],
```
Comment (tonybaloney) [inline src/common/names.ts:329]: Please remove "My Python"

File: src/common/names.ts [hunk 4]
```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
     [8, 'Bailey'],
     [9, 'Max'],
     [10, 'Seb'],
+    [11, 'Mr.Hiss'],
+    [12, 'Miss.Hiss'],
```
Comment (tonybaloney) [inline src/common/names.ts:324]: Please be consistent with the space after the `Mr.` and `Mrs.` I think they should all just have a space, e.g. `Mr Hiss`

File: src/common/names.ts [hunk 5]
```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
...
+    [20, 'Steelix'],
+    [21, 'Gyarados'],
+    [22, 'SandaConda'],
+    [23, 'Seviper'],
+    [24, 'Ekanes'],
+    [25, 'Arbok'],
+    [26, 'Snivy'],
+    [27, 'Servine'],
+    [28, 'Serperior'],
+    [29, 'Hiss Speed'],
```
Comment (tonybaloney) [inline src/common/names.ts:341]: I don't understand this one

File: src/common/names.ts [hunk 6]
```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
...
+    [13, 'Snaku'],
+    [14, 'Kaa'],
+    [15, 'Madame Snake'],
+    [16, 'Sir Hiss'],
+    [17, 'My Python'],
+    [18, 'The Anaconda'],
+    [19, 'Loki'],
+    [20, 'Steelix'],
+    [21, 'Gyarados'],
+    [22, 'SandaConda'],
```
Comment (tonybaloney) [inline src/common/names.ts:334]: Missing a space

File: src/common/names.ts [hunk 7]
```diff
@@ -338,6 +358,30 @@ export const DUCK_NAMES: Map<number, string> = new Map<number, string>([
     [14, 'Quackers'],
     [15, 'Jelly Beans'],
     [16, 'Donald'],
+    [17, 'Psyduck'],
+    [18, 'Waddles'],
+    [19, 'Mr.Quack'],
```
Comment (tonybaloney) [inline src/common/names.ts:363]: Same as above re: spacing

File: src/common/names.ts [hunk 8]
```diff
@@ -368,6 +415,17 @@ export const ROCKY_NAMES: Map<number, string> = new Map<number, string>([
...
     [13, 'Rockstar'],
+    [14, 'Onix'],
+    [15, 'Rock and Roll'],
+    [16, 'Dolomite'],
+    [17, 'Granite'],
+    [18, 'Miss Marble'],
+    [19, 'Rock On'],
+    [20, 'Amberstone'],
+    [21, 'Rock With Me'],
+    [22, 'Rock On It!'],
```
Comment (tonybaloney) [inline src/common/names.ts:426]: Please remove the exclamation mark

**Review by tonybaloney — CHANGES_REQUESTED** [review]

**Review by ShabanaNaik — COMMENTED** [review]
I have made the changes as requested. Thank you for your patience.

**Comment by ShabanaNaik** [comment]
I have noticed this PR has conflicts with the real file that I am unable to understand. I would like to close this and open a new PR if that would possible after making necessary changes.

**Pull request closed without merging**